composer.json

```diff
@@ -146,8 +147,8 @@
         "drupal/views_block_filter_block": "^1.0@beta",
         "drupal/views_bulk_operations": "^3.6",
         "drupal/xmlsitemap": "^1.0",
-        "ext-imagick": "*",
         "onlyextart/colorbox": "dev-master#e58476becbc89dc671093d1bcd9f99b2167fa8f7",
+        "su-sws/drupal-patches": "^8.0",
```

Comment:
Cool. It will be interesting if/how this gets versioned when patches get rolled and our different teams keep up at different paces.

Any thought to ditching patches altogether and instead, running forks?

Reply:
I think forks is going to be really tough for those instances that we need 2 or more patches. And patches keep the rest of the module up to date so it's tough to argue in favor of forks at this time.